packages/itwinui-css/backstop/tests/checkbox.html

```diff
@@ -571,7 +571,9 @@ <h3>Checkbox on left</h3>
             tabindex="-1"
           />
           <div
-            class="iui-progress-indicator-radial iui-indeterminate"
+            class="iui-progress-indicator"
+            data-iui-variant="radial"
+            data-iui-indeterminate="true"
```

Comment:
I'm liking this new setup for the progress indicator display. Allowing cutsomization through `data` types instead of just a `class` will make it a lot more efficient to switch between different display varieties.

Reply:
I do much prefer using data-attributes over class modifiers. ☺️